.github/scripts/end2end/install-kind-dependencies.sh

```diff
@@ -99,17 +99,18 @@ build_solution_base_manifests() {
     # Limits and requests for MongoDB are computed based on the current system
     # Detect total system RAM in GiB
     TOTAL_RAM_GB=$(awk '/MemTotal/ {printf "%.0f", $2/1024/1024}' /proc/meminfo)
+  
     # Compute MongoDB settings based on the total RAM
-    MONGODB_WIREDTIGER_CACHE_SIZE_GB=$(awk "BEGIN {printf \"%.0f\", ${TOTAL_RAM_GB} * 0.335}")
-    MONGODB_MONGOS_RAM_LIMIT=$(awk "BEGIN {printf \"%.0fGi\", ${TOTAL_RAM_GB} * 0.165}")
-    MONGODB_SHARDSERVER_RAM_SAFE_LIMIT=$(awk "BEGIN {printf \"%.0fGi\", 2 * ${MONGODB_WIREDTIGER_CACHE_SIZE_GB}}")
-    MONGODB_SHARDSERVER_RAM_REQUEST=${MONGODB_WIREDTIGER_CACHE_SIZE_GB}Gi
-    MONGODB_MONGOS_RAM_REQUEST=$(awk "BEGIN {printf \"%.0fGi\", ${TOTAL_RAM_GB} * 0.033}")
+    MONGODB_SHARDSERVER_EXTRA_FLAGS=$((TOTAL_RAM_GB * 335 / 1000))
+    MONGODB_MONGOS_RAM_LIMIT=$((TOTAL_RAM_GB * 165 / 1000))Gi
+    MONGODB_SHARDSERVER_RAM_LIMIT=$((2 * MONGODB_SHARDSERVER_EXTRA_FLAGS))Gi
+    MONGODB_SHARDSERVER_RAM_REQUEST=${MONGODB_SHARDSERVER_EXTRA_FLAGS}Gi
+    MONGODB_MONGOS_RAM_REQUEST=$((TOTAL_RAM_GB * 33 / 1000))Gi
 
     # Replace values before deploying
-    sed -i "s/MONGODB_WIREDTIGER_CACHE_SIZE_GB/${MONGODB_WIREDTIGER_CACHE_SIZE_GB}/g" $DIR/_build/root/deploy/*
+    sed -i "s/MONGODB_SHARDSERVER_EXTRA_FLAGS/--wiredTigerCacheSizeGB=${MONGODB_SHARDSERVER_EXTRA_FLAGS}/g" $DIR/_build/root/deploy/*
```

Comment:
here as well:

```
sed -i "s/MONGODB_SHARDSERVER_EXTRA_FLAGS/--wiredTigerCacheSizeGB=${MONGODB_WIRETIGER_CACHE_SIZE_GB}/g" $DIR/_build/root/deploy/*
```

Reply:
Thanks, done in https://github.com/scality/Zenko/pull/1957/commits/45ea576761c41ec4c5d2ecb4610cb6ef0f27c5be